tests/common/fixtures/duthost_utils.py

```diff
@@ -698,10 +698,11 @@ def convert_and_restore_config_db_to_ipv6_only(duthosts):
                     finally:
                         ssh_client.close()
 
-        pytest_assert(len(ipv6_address[duthost.hostname]) > 0,
-                      f"{duthost.hostname} doesn't have IPv6 Management IP address")
-        pytest_assert(has_available_ipv6_addr,
-                      f"{duthost.hostname} doesn't have available IPv6 Management IP address")
+        if not has_available_ipv6_addr:
+            pytest.skip(f"{duthost.hostname} doesn't have available IPv6 Management IP address")
+
+        if not ipv6_address[duthost.hostname]:
+            pytest.skip(f"{duthost.hostname} doesn't have IPv6 Management IP address")
```

Comment:
Can you elaborate on why reordering these 2 conditions?

If we do have assigned IPv6 address but not configured on the mgmt port, should this be a failure or skip?

Reply:
Hi @yxieca for me it still setup issue and not TC issue, so I'd prefer skip rather than fail. How do you think?

Reply:
@SavchukRomanLv I think Ying's question is why you check `has_available_ipv6_addr` before `ipv6_address`. Is there a particular reason for changing the order?